frontend/workflows/projectCatalog/src/index.tsx

```diff
@@ -1,9 +1,15 @@
 import type { BaseWorkflowProps, WorkflowConfiguration } from "@clutch-sh/core";
 
+import type { DetailsCard } from "./details/card";
+import type { ProjectInfoChip } from "./details/info/chipsRow";
 import Catalog from "./catalog";
 import Details from "./details";
 
 export interface WorkflowProps extends BaseWorkflowProps {}
+export interface DetailWorkflowProps {
```

Comment:
nit: this name should probably match with the export name `ProjectDetailsWorkflowProps`

Reply:
Good call